python/ray/data/tests/test_dataset_formats.py

```diff
@@ -932,15 +932,26 @@ def test_parquet_read_parallel_meta_fetch(ray_start_regular_shared, fs, data_pat
 
 
 def test_parquet_reader_estimate_data_size(shutdown_only, tmp_path):
```

Comment:
Could we also add an integration test that the size estimate is reflected in `ds.size_bytes()` properly?

Reply:
@ericl - sure, added.